app/src/main/java/com/mk/tasky/agenda/presentation/detail/event/DetailEventViewModel.kt

```diff
@@ -163,6 +172,61 @@ class DetailEventViewModel @Inject constructor(
                     selectedFilterType = event.type
                 )
             }
+            is DetailEventEvents.OnAddVisitor -> {
+                if (validateEmailUseCase(state.dialogEmail)) {
+                    viewModelScope.launch {
+                        state = state.copy(
+                            isLoadingDialog = true
+                        )
+                        eventUseCases.getAttendee(state.dialogEmail).onSuccess {
+                            it?.let {
+                                if (state.attendees.firstOrNull { element -> element.userId == it.userId } == null) {
```

Comment:
Might be a bit nitpicky, but I personally prefer to save such lambda results in a variable first, since otherwise it's hard to scan this piece of code as you need to use your brain to understand what this expression evaluates to. If you save it in a variable `doesAttendeeExist`, you don't need a second to understand when the code jumps into this if-block

Reply:
Sounds good. Just pushed the change!